trivia-app/client/src/components/Question.js

```diff
@@ -0,0 +1,64 @@
+import AnswerChoices from "./AnswerChoices";
+const Question = ({
+  amount,
+  result,
+  score,
+  setScore,
+  questionNum,
+  setQuestionNumber,
+}) => {
+  /**
+   * Source: https://linuxhint.com/decode-html-entities-javascript/
+   * Decode HTML Entities
+   * @param {event} e
+   */
+  function decode(str) {
```

Comment:
Would you mind pulling out this code outsidre the component? When a function doesn't use something from a component we can declare it outside the component.